data_generation/ISO3.py

```diff
@@ -59,6 +70,80 @@ def main():
     return climate_categories.HierarchicalCategorization.from_spec(spec)
 
 
+def add_basic(categories):
+    categories["BASIC"] = {
+        "title": "BASIC countries",
+        "children": [["BRA", "ZAF", "IND", "CHN"]],
+    }
+    return categories
+
+
+def add_ldc(categories):
+    categories["LDC"] = {
+        "title": "Least Developed Countries",
+        "children": [
+            [
+                "AGO",
+                "BEN",
+                "BFA",
+                "BDI",
+                "CAF",
+                "TCD",
+                "COM",
+                "COD",
+                "DJI",
+                "ERI",
+                "ETH",
+                "GMB",
+                "GIN",
+                "GNB",
+                "LSO",
+                "LBR",
+                "MDG",
+                "MWI",
+                "MLI",
+                "MRT",
+                "MOZ",
+                "NER",
+                "RWA",
+                "SEN",
+                "SLE",
+                "SOM",
+                "SSD",
+                "SDN",
+                "TGO",
+                "UGA",
+                "TZA",
+                "ZMB",
+                "AFG",
+                "BGD",
+                "KHM",
+                "LAO",
+                "MMR",
+                "NPL",
+                "TLS",
+                "YEM",
+                "HTI",
+                "KIR",
+                "SLB",
+                "TUV",
+            ]
+        ],
+    }
+    return categories
+
+
+def add_umbrella(categories):
+    categories["UMBRELLA_2022"] = {
+        "title": "Informal coalition that likes to take a common position during climate negotiations",
+        "comment": "Following the 2022 invasion, Russia and Belarus were ejected from the group",
+        "children": [
+            ["KAZ", "ISR", "AUS", "ISL", "NZL", "USA", "JPN", "CAN", "NOR", "ISR"]
+        ],
+    }
```

Comment:
```suggestion
    categories["UMBRELLA_2022"] = {
        "title": "The Umbrella Group is a coalition of Parties which formed following the adoption of the Kyoto Protocol",
        "comment": "Following the 2022 invasion, Russia and Belarus were ejected from the group",
        "children": [
            ["KAZ", "ISR", "AUS", "ISL", "NZL", "USA", "JPN", "CAN", "NOR", "ISR"]
        ],
    }
    categories["UMBRELLA_2023"] = {
        "title": "The Umbrella Group is a coalition of Parties which formed following the adoption of the Kyoto Protocol",
        "comment": "The United Kingdom formally joined the group in 2023.",
        "children": [
            ["AUS", "CAN", "ISL", "ISR", "JPN", "NZL", "KAZ", "NOR", "UKR", "USA", "GBR"]
        ],
        alternative_codes: ["UMBRELLA"],
    }
```

This is the most up to date list I could find. I'm wondering where you got the members in 2022 from, because it doesn't include UKR.

I also added the "UMBRELLA" code for the most recent entry.

Reply:
ok thanks!
I don't remember actually. Wikpedia probably. I guess I forgot to add UKR

Reply:
If we're not sure when exactly UKR acceded, let's just remove the UMBRELLA_2022 definition? I guess folks are mostly interested about the most recent version and we don't need to spend time researching it's history.

Reply:
Yeah maybe that's better.